### ISSUE TITLE
[FEATURE REQUEST] Change email to non gmail one

### ISSUE THREAD

**Issue by darigovresearch** [gosqasorg/asset-provenance-tracking#69]
**Is your feature request related to a problem? Please describe.**
Change email in profile to a non-gmail one

**Describe the solution you'd like**
If you have the gosqas domain setup a hello@gosqas.org

**Describe alternatives you've considered**
There are other good email providers too

**Additional context**
![image](https://github.com/gosqasorg/asset-provenance-tracking/assets/30328618/559318ad-239b-4e06-a275-6256133852d2)

**Comment by devhawk** [comment]
@devhawk to look into what email services the MSFT Not-for-profit grant provides

**Comment by devhawk** [comment]
The MSFT Nonprofit grant we received includes [10 free Microsoft 365 Business Premium accounts](https://www.microsoft.com/en-us/nonprofits/offers-for-nonprofits?rtc=1#primaryR3), which includes email. I will investigate what it takes to set up an @gosqas.com email server. I assume we won't need more than ten accounts (webmaster, admin, help, info ???)

**Comment by RobertLRead** [comment]
Yes, we don't need more that 10 accounts. Note this may be somewhat similar to the fact that we need to send email from the server as part of the notification system. @Htauqee is working on that; you may need to confer with her on that.

**Comment by devhawk** [comment]
Using MSFT 365 Business Premium Account for this seems like massive overkill. That service is designed for end users so it comes with the full office suite, a TB of storage, InTune Management, Sharepoint, and a partridge in a pear tree *plus* email services. I'd suggest that we don't want to take the time and effort to setup and manage MSFT 365 for this scenario. Unless we want to give real team members a @gosqas.org address (i.e. harry@gosqas.org) I vote that we don't do this.

Squarespace provides a free [email forwarding service](https://support.squarespace.com/hc/en-us/articles/19000909092237-Email-forwarding-with-a-Squarespace-domain). Basically, we can create up to 100 @gosqas.org websites that get forwarded to whatever address we want. We could easily forward `hello@gosqas.org` to the existing `gosqasystem@gmail.com` address. Would take just a few minutes to create the forwarding alias plus whatever time needed for a front end engineer to update the email address on the existing website.

**Comment by vjaqua** [comment]
Whereever gosqasystem@gmail.com appears, update it to info@gosqas.org @agilpatr
